os/android/libhwcservice/hwcserviceapi.h

```diff
@@ -127,23 +127,6 @@ status_t HwcService_Display_SetDeinterlaceParam(HWCSHANDLE hwcs,
 
 // DisplayModeControl
 
-typedef enum _EHwcsModeFlags {
-  HWCS_MODE_FLAG_NONE = 0,
-  HWCS_MODE_FLAG_PREFERRED = 1 << 0,
-  HWCS_MODE_FLAG_SECURE = 1 << 1,
-  HWCS_MODE_FLAG_INTERLACED = 1 << 2,
-  HWCS_MODE_FLAG_CURRENT = 1 << 4,
-} EHwcsModeFlags;
-
-/// Enumerations for common aspect ratios
-/// Any ratio can be supported, with the upper 16 bits containing one dimension,
-/// the lower 16 bits contains the lower dimension
-typedef enum _EHwcsModeAspectRatio {
-  HWCS_MODE_ASPECT_RATIO_ANY = 0x00000000,
-  HWCS_MODE_ASPECT_RATIO_4_3 = 0x00040003,
-  HWCS_MODE_ASPECT_RATIO_16_9 = 0x00100009,
-} EHwcsModeAspectRatio;
-
```

Comment:
Why is this block getting removed? Are you cleaning up stale code? If so, could we put this in a separate patch?

Reply:
dropped from here